docs/AboutUs.md

```diff
@@ -29,14 +29,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Team Lead
 * Responsibilities: UI
 
-### Johnny Doe
+### Glemen Neo
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/glemenneo.png" width="200px">
 
-[[github](http://github.com/johndoe)] [[portfolio](team/johndoe.md)]
+[[github](http://github.com/glemenneo)] [[portfolio](team/johndoe.md)]
 
 * Role: Developer
-* Responsibilities: Data
+* Responsibilities: Testing
```

Comment:
I believe Testing is a Role as stated on the module website; you can elaborate more under Responsibilities

Reply:
I have changed the Role accordingly and added more elaboration under Responsibilities. Thanks for the heads up.